### PR TITLE
Multiple RPC providers

### PR DESCRIPTION
**Pull request by jvonasek**

### CODE REVIEW

File: src/components/Toast/Toast.utils.ts [hunk 1]
```diff
@@ -230,7 +230,10 @@ export const useBridgeToast = (toasts: ToastData[]) => {
                 bridge: "ethereum",
               })
             }
-          } catch (error) {}
+          } catch (error) {
+            toast.remove(toastData.id)
```
Comment (vkulinich-cl) [inline src/components/Toast/Toast.utils.ts:234]: Remove this please

File: src/sections/provider/ProviderSelectModal.tsx [hunk 2]
```diff
@@ -1,192 +1,97 @@
...
   return (
     <>
       <Modal
         open={props.open}
         onClose={props.onClose}
         title={t("rpc.change.modal.title")}
+        headerVariant="FontOver"
       >
-        <>
-          {import.meta.env.VITE_ENV !== "production" && (
```
Comment (vkulinich-cl) [inline src/sections/provider/ProviderSelectModal.tsx:66]: Have you removed that conditional?

File: src/components/Toast/Toast.utils.ts (hunk 1 above)
Comment (jvonasek) [reply src/components/Toast/Toast.utils.ts]: removed

File: src/sections/provider/ProviderReloader.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { useQueryClient } from "@tanstack/react-query"
+import { useActiveRpcUrlList } from "api/provider"
+import { useRemount } from "hooks/useRemount"
+import { Fragment, useEffect, useState } from "react"
+import { usePrevious } from "react-use"
+import { ProviderSelectButton } from "sections/provider/components/ProviderSelectButton/ProviderSelectButton"
+
+export const ProviderReloader: React.FC<{ children?: React.ReactNode }> = ({
```
[the remark this answers is not in the document]
Comment (jvonasek) [reply src/sections/provider/ProviderReloader.tsx]: It doesn't but there is PropsWithChildren instead

File: src/sections/provider/ProviderSelectModal.tsx (hunk 2 above)
Comment (jvonasek) [reply src/sections/provider/ProviderSelectModal.tsx:66]: Yes, user can add his own RPC even on production now.

**Review by vkulinich-cl — APPROVED** [review]

**Pull request merged**